src/main.rs

```diff
@@ -77,7 +83,18 @@ fn main() {
                 .takes_value(true)
                 .value_name("FILE")
                 .default_value("cache.zip")
-                .help("Specify where the created cache file should be saved.")));
+                .help("Specify where the created cache file should be saved."));
+    
+    #[cfg(feature = "cover")]
+    let cache_command = cache_command.arg(Arg::with_name("coverdb")
```

Comment:
I would default this to true and instead have a `--no-cover-db` option that prevents the generation.

Reply:
sure